productionConfig

```diff
@@ -0,0 +1,3 @@
+zrlfyamblttpom
+e2c0e8832ea228e6b15e553ce69f7cb2c0ff4d646ff0f284245ce77cc78b437b
+d7ckgvm53enhum
```

Comment:
Please, use a simple JSON format for your config data. You could read it with a standard json.Unmarshal, have a struct for configuration, and read it easily